lib/src/visitor/async_evaluate.dart

```diff
@@ -512,7 +539,70 @@ final class _EvaluateVisitor
             configuration: configuration,
             namesInErrors: true);
         _assertConfigurationIsEmpty(configuration, nameInError: true);
-      }, url: "sass:meta")
+      }, url: "sass:meta"),
+      BuiltInCallable.mixin("apply", r"$mixin, $args...", (arguments) async {
+        var mixin = arguments[0];
+        var args = arguments[1] as SassArgumentList;
+
+        var callableNode = _callableNode!;
+        var invocation = ArgumentInvocation([], {}, callableNode.span,
+            rest: ValueExpression(args, callableNode.span),
+            keywordRest: args.keywords.isEmpty
+                ? null
+                : ValueExpression(
+                    SassMap({
+                      for (var (name, value) in args.keywords.pairs)
+                        SassString(name, quotes: false): value
+                    }),
+                    callableNode.span));
+
+        var callable = mixin.assertMixin("mixin").callable;
+        // ignore: unnecessary_type_check
+        if (callable is AsyncCallable) {
+          switch (callable) {
+            case BuiltInCallable() when _environment.content != null:
+              throw _exception("Mixin doesn't accept a content block.");
+
+            case BuiltInCallable():
+              await _runBuiltInCallable(invocation, callable, callableNode);
+
+            case UserDefinedCallable<AsyncEnvironment>(
+                  declaration: MixinRule(hasContent: false)
+                )
+                when _environment.content != null:
+              throw MultiSpanSassRuntimeException(
+                  "Mixin doesn't accept a content block.",
+                  callableNode.span,
+                  "invocation",
+                  {callable.declaration.arguments.spanWithName: "declaration"},
+                  _stackTrace(callableNode.span));
+
+            case UserDefinedCallable<AsyncEnvironment>():
+              var contentCallable = _environment.content.andThen((content) =>
+                  UserDefinedCallable(
+                      content as CallableDeclaration, _environment.closure(),
+                      inDependency: _inDependency));
```

Comment:
`_environment.content` is already a `UserDefinedCallable`. Why not just use that as-is? In fact, do you need to redefine this at all, or will the existing `_environment.content` just naturally fall through to the child mixin?